src/wallet/coinselection.h

```diff
@@ -56,31 +57,58 @@ class CInputCoin {
     }
 };
 
+/** Parameters for filtering which OutputGroups we may use in coin selection.
+ * We start by being very selective and requiring multiple confirmations and
+ * then get more permissive if we cannot fund the transaction. */
 struct CoinEligibilityFilter
 {
+    /** Minimum number of confirmations for outputs that we sent to ourselves.
```

Comment:
Useful to clarify what kinds of outputs are sent to ourselves.

```suggestion
    /** Minimum number of confirmations for outputs that we sent to ourselves (e.g. change outputs).
```